.config/ruff.toml

```diff
@@ -9,7 +9,10 @@ select = [
   "I"    # isort
 ]
 ignore = [
-    "E501" # Line too long
+    "E501", # Line too long
+    "D100", # missing-module-docstring
+    "D101", # missing-class-docstring
+    "D103" # missing-function-docstring
```

Comment:
Why turn this off, do you not like writing docstrings Ian? :D

Reply:
The last project I worked on they fanatically excluded comments and docstrings from code! In any case most the classes in this project lack docstrings.

Typically on line length I set something longer than the default (usually 100 characters) and then comply with it pretty rigorously.

Reply:
Hm good point that it's not catching the missed class docstrings, I will investigate